src/Dropdown/ButtonTrigger.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+export default class ButtonTrigger extends Component {
```

Comment:
Why is the `InputTrigger` and DropdownTriggers.js` file gone?

*Just a suggestion*: what if you make a base class `DropdownTrigger` that is the "interface" a component needs in order to be trigger. And then let `ButtonTrigger`, `InputTrigger`, etc extend this class (and be in the dropdown triggers file?). Right now this PR allows "any tag as a trigger", but I think it should enforce the behavior of trigger components.

Reply:
yeah, we discussed this yesterday and thought it will be good enough having a `ButtonTrigger` - but you are right, we should implement an abstract `DropdownTrigger` 👍  
@nLight & @orlandohohmeier are discussing it further, lets see what their conclusion will be.